libraries/AP_Winch/AP_Winch.cpp

```diff
@@ -101,6 +102,56 @@ void AP_Winch::set_desired_rate(float rate)
     config.control_mode = ControlMode::RATE;
 }
 
+//Implement DO_WINCH Deliver message
+void AP_Winch::package_update()
+{
+    if ( deliveryFlag == true)
```

Comment:
nitpick: style is a little off here.  no spaces after the "(" and curly brace should be on the same line.